pickle.c

```diff
@@ -2491,7 +2499,7 @@ int pickle_delete(pickle_t *i) {
 	return r != PICKLE_OK ? r : PICKLE_OK;
 }
 
-#ifdef NDEBUG
+#if DEFINE_TESTS
 int pickle_tests(void) { return PICKLE_OK; }
```

Comment:
Other way around?

Reply:
Ooops. That condition unfortunately got reversed.
I am also in favor of the C language if() instead of the preprocessor. That way the syntax checking is performed even when the functionality is disabled.